internal/security/secretstore/init.go

```diff
@@ -49,6 +50,13 @@ import (
 	"github.com/edgexfoundry/go-mod-secrets/v2/secrets"
 )
 
+const (
+	addKnownSecretsEnv = "ADD_KNOWN_SECRETS"
+	redisSecretName    = "redisdb"
+)
+
+var validKnownSecrets = map[string]bool{redisSecretName: true}
```

Comment:
please avoid using the global variable, put into calling object instance if you can

Reply:
Technically this isn't a global variable since it is only known to the package, but since this service all in one package... it is a basically a global. ;-) 

Moved to the Bootstrap struct

Reply:
nice! i like this much better